.github/workflows/node.js.yml

```diff
@@ -0,0 +1,20 @@
+name: Test.js CI
```

Comment:
This might work but then is not part of the requirements. This setup nodejs app. There are three GitHub actions required: 
- Github actions for gh pages deployment
- automated testing for unit test
- automated testing for e2e

Reply:
Should now be fixed